### ISSUE TITLE
Milestone 2 feedback

### ISSUE THREAD

**Issue by kandarpksk** [AKBarcenas/cogs-121#1]

**Comment by ConnieGuan** [comment]
Comments from Team YellowCow
Your First story board: Event Finder
I like the idea of finding events around someone, but I question what the application of maps/geography is. This looks almost like a craigslist posting where people can post events and ask other people to join. It doesn't seem like it's uniquely grounded in a physical location.

Your Second story board: Heatmap
The idea of local news is also pretty cool! But I still wonder how this app offers anything different than going to google for San Diego and clicking on trending news. Or even just looking at the facebook trending news. Maybe trying to find a way to connect it more to physical location or maps would help it offer something.

**Comment by streddy** [comment]
Team: SoDoge (K5)

We like the last storyboard idea. It would be cool if you could implement some sort of voting aspect to have all members of a group input their preferences and then have the app spit out various options that are close to them. 

The local trending topics is also a cool idea, but isn't this already implemented a little in Facebook/Twitter? Maybe I'm wrong but I thought that there were some features that showed more localized trending topics. How would you define a "local" region? 

The microclimates one is pretty cool (great minds think alike haha). But this is definitely something that happens on campus a lot so you tapped into a need that a lot of people could relate to. We think it could be useful for students on campus/people in the greater area.

**Comment by jorennejf** [comment]
Food Place Locater Chatbot Storyboards #1: 
I'm confused as to whether the chat bot notifies you or you have to open up the app and then talk to the chat bot.
Is it voice controlled?
I think it's only one person using the app to talk to the chat bot (#ForeverAlone) so I'm guessing the situation is one person finding a place to eat.

Food Place Locater Chatbot Storyboards #2: 
This one uses text-based recommendations 
It's a little confusing understanding how the app resolves the situation, since there are many people with different food preferences but what exactly does the app do to resolve this?

Temperature Map Storyboard #1:
I really like this idea, but what makes it different from checking the weather on the weather app?
The situation is understandable; I too have to plan my outfits based on weather. I like the idea, but maybe find a way to make it more unique than other weather apps.

TritonPark Storyboard #1:
DEF A MUCH NEEDED APP, but how do you go about getting the data for this?
Is it crowd-sourced?
Good idea, but maybe expand it to other locations outside of UCSD? (unless this is too hard and the TA is okay with having it just work at UCSD)

**Comment by reesewahlin** [comment]
Storyboard 1 (event finder):

Good storyboard with a clear setting, sequence, and result.

A cool addition would be some sort of confirmation system to see all the people currently attending so when they go to the frisbee event tomorrow they know for sure the event is still going on and they have the numbers to do so.

**Comment by meschneider** [comment]
Feedback for sanjeev_park.jpg: 

Your storyboard is well organized and has a situation, sequence, and an end result.

I think this idea is really useful, especially with the parking situation at UCSD. While reading this I wondered, how will you be able to track which spots are open? I think this idea may be too complicated or difficult to implement.

**Comment by chadbet** [comment]
Team c8:

Food place locator storyboard 1:
It seems like this idea has already been manifested in other applications such as Yelp which allow a user to customize a search based off what they are hungry for and then use crowd sourced ratings to determine what is good. A possible option to differentiate is to aim this at friends - meaning I am more likely to choose to grab food at a restaurant if a friend personally told me that it is good. The storyboard could be better at explaining what the grabbing feature is outside of the map. 

Storyboard 2:
This is an interesting idea but I think it isn't focusing on what the main issue with deciding where to go with friends is: opinions. Maybe instead of looking for a food place that satisfies X number of categories you could input what everyone wants, pick an area on the map, and then it essentially randomly selects a place that fits most criteria in an area that you want. Again, anything food locator related has been handled pretty well by Yelp, Google, and UrbanSpoon so I would suggest either looking to them for inspiration on what they don't do well or pivoting to a different idea.

**Comment by meschneider** [comment]
Feedback for sanjeev_heatmap.jpg: 

Your storyboard is well organized and has a situation, sequence, and an end result.

I like your idea, but I think a lot of news websites already have a section such as "popular near you". How will you make your application different than these? What kind of news will be featured? One idea you could implement is to have news from various perspectives, which many news sites do not have. I think it would be interesting to have liberal and conservative pieces of news together for example.

**Comment by reesewahlin** [comment]
Storyboard 2 (event finder):

Good setting, sequence, and result.

One issue I foresee arising is for invites where the number of players is fixed. Say you're trying to get a basketball game going and 15 people end up coming, while only 5 can play on either team. There is going to be a group waiting and not all will get playing time. Some sort of reservation system may be ensure something like this doesn't happen, but then you run into the issue of no shows offsetting the sizes of the teams.

**Comment by NickCampbellUCSD** [comment]
Team Guography

**Feedback for "albert_heatmap.jpg" (third storyboard) and
Feedback for sanjeev_heatmap.jpg**
I like the concept of locally trending news, the difficulty is going to be adding functionality to features that already exist in popular existing websites. Specifically, Twitter offers trending hashtags, Twitter moments, and trending news, and following people can give you news instantaneously, news you prefer and that you decide is relevant. In order to stand out, maybe like making the news super local - the use here of a fire alarm is good -  and utilize sources of information that media like Twitter won't be able to.

The second storyboard is similar to the first but again, there should be a differentiating factor between this app and something like Twitter. How are you guys adding utility or resolving a breakdown in the way that people consume news?

**Comment by agao323** [comment]
## Team ABG

### Storyboard 3 (Albert Lin)
* The correlation behind the problem and the solution is clear
* The solution in this situation seems a little odd, in that someone would go to a completely different location to find out what was going on where the people were talking about the "big important news"
* Make sure to think about how you would actually implement this; from the storyboard it is not clear, and it almost makes it seem like you are going to somehow try and figure out what news stories are trending based on what people are saying. You could easily pull from Facebook or Twitter to identify trending news stories though, I like the idea

### Storyboard 6 (Sanjeev Reddy)
* One problem with this is that every single team I have reviewed so far has had this idea to help people find parking. If you go with this idea, try to find some way to make it unique or different.
* Another issue I have with the storyboard is that there is no way of knowing how many parking spots are left, unless you install sensors or something across the entire parking structure. 
  * Even then, there is no guarantee that those parking spots will still be there when you get there, unless you change UCSD policy and only allow people to park for a certain length of time, which doesn't seem very feasible
  * How would you know that there are 45 spots available in 30 minutes if there are only 30 spots available currently?

**Comment by liawesomesaucer** [comment]
## Team ABG (16) (C7)

### Overall feedback
* Some of your storyboards are only 4 or 5 panes and could benefit a lot from having an extra pane or two describing the situation or steps in how your solution solves the problem
* Some of these ideas do have potential but others (like parking or basketball event-finding) have been well explored in project classes already. Make sure if you're going down this route to explore the problem in a novel way
* Make sure to think of how you will source data for certain problems (ie what incentive is there for people to provide parking/temperature/food desirability data?)

### Storyboard 8  (Finding a food place for a group that meets everyones needs (Alex Barcenas))
* This problem definitely exists and is quite commonly used. However using a chat interface when people are already in person seems slightly cumbersome - how would this be noticeably different from using a polling application + maybe Yelp? Also how do you handle disagreements - do you just continue down a list of proposed food places?

### Storyboard 5 (Finding micro-climates/temperatures (Albert Lin))
* This idea seems cool and technically feasible, if you have the data. But I'm not sure how you'd collect data for this as you'd need temperature measurements every two blocks. In addition, the highest variability in temperature would probably be from transitioning between indoor/outdoor, which is probably more difficult to gather (you'd need access to thermostats)

### Storyboard 2 (Hosting Events to gather people of similar interest near me (Todd Tang) )
* This fits in the realm of event-finding applications, which is done quite often. That said, there's probably a definite need for this but would require critical mass to become fully effective

**Comment by rdhanaraj** [comment]
_**Team ABG**_

**Storyboard 1:**
You want to make sure this storyboard reflects the use case you want to address. Right now, I'm not sure it does, because the problem you pose in the storyboard is that you want to find "similar events" to events the user has already done. However, looking at the third panel, I'd guess you want users to find nearby events, even if they haven't been to some events before.

You should try to narrow down this use case. What kinds of events are people looking for? How do you know what kinds of events they're looking for? etc.

tl;dr very broad, what events is this app for, what is the specific use case

**Storyboard 4:**
This idea could definitely be pretty cool. However, the value the user would get out of this kind of app would NOT be becoming a more informed member of society. The value should have something to do with community, or local news.

Also, wondering about news popular near me doesn't sound like a problem. Wanting to be educated so you can have cool conversations with your friends, sounds more like an actual problem (imo). Basically, make sure you can identify what the need is there.

tl;dr pinpoint specific need, specific value

**Storyboard 7:**
This storyboard is good in that it has a clear need and a clear solution. I wonder how you would be able to give good food recommendations to people near them, but I suppose implementation is left out of storyboards.

It's interesting the user ends up eating a specific food, ramen in the last panel. Would the app guess what the user is craving? If you could specify that, that would be useful.

**Comment by cjiao12** [comment]
Team 18
The event-find app is a pretty cool idea
For local trending topics, what is stopping the user from checking other more available sites like Facebook, Twitter, reddit, etc.
Overall the storyboards are pretty clear in what the use of your app ideas will be, however I think some of your ideas are a little vague or don't stand out from easily accessible other apps that already exist

**Comment by dssung** [comment]
EventFinder
- Could be done with FB. I think it's too general. More specific activities would be better. 

HeatMap
- I like this idea a lot. It reminds me of Yik Yak but location based and visual.

Temperature Map
- It might be tough to acquire microclimates on campus, unless you crowdsource.

Triton Park
- It will be difficult to implement because it's hard to get parking lot data, unless you again crowdsource.

Food Chat Bot
- I like the idea. I think chat bots make the app easy and fun to use. 

Second Food Idea
- Doesn't really incorporate a map. Can be easily done just by texting/messaging.

**Comment by grantnelson** [comment]
Team 18 (c9) Grant:

For the first two event storyboards, I thought that they had pretty overlapping ideas. The idea of a nice events app is always valid, but in my opinion pretty difficult to pull off. You might want to focus on a specific facet of the idea such as a single type of event or demographic(maybe students). Perhaps you can pull from a api of facebook or eventbrite. I think that you should try to slim the idea down to a more specific topic. A broad event app will not be feasible. 

The Temperature map storyboard was a really well thought out idea. I think that weather apps are too total encompassing and would be better if they had a heat map like you suggested. Maybe even ignoring other weather patterns such as rain etc even- or just there as an afterthought. I think that your idea will have potential if you can make the interface inviting and easy to read. Not too many pages or complex features

**Comment by nonobitata** [comment]
Team 11 ( Kool Kats) - Trung:
Storyboard 1: Finding Events of interest near me
- Should user look up for a keyword ("frisbee" / "soccer") to find an event of interest or they just have a map that shows all events around them? This may narrow down what are use cases of the app.
Storyboard 2:
- Same as storyboard 1.
Storyboard 3: Finding local trending topics
- What sources are you using to show news in the area? Is it from newspaper online or from user's reports? This will define your app as a "local newspaper" or "local social networks".

**Comment by lfpadill** [comment]
Team 18
I really like the app for finding a restaurant for a group. That is my favorite as I see this as being useful.  I also like the event planner and event finder app. I think they could be combined into one app. The temperature idea seems really unique and useful if you live on campus. I do think however that expanding it would be much more helpful. Finally, the parking app seems very practical but the idea is popular among other teams so maybe one of your other ideas would be better.

**Comment by jphung11** [comment]
Team #11:

Storyboard 4:
The storyboard is concise and clear. As for the idea, how would this be different from watching your local news or going on your local news website/app. We think this idea could have a lot of potential if more features could be added to differentiate it from an app that just gets local news.

Storyboard 5:
The idea is very interesting and cool. One thing to consider is to the possibility of actually making the implementation within less than 10 weeks. However, if your team could do it, we think this idea is great and has lots of potential.

Storyboard 6: 
Good idea but it seems like it is only limited to UCSD. It would be cool to make it more general and apply it to other public parking spaces. In addition, one thing to consider is whether the feedback from the app would be live or not. This is with the consideration that spots can be taken on the way over to the parking lot.

Storyboard 7:
This could be a very beneficial app idea, especially when one is indecisive or unaware of what an area has to offer. The storyboard was concise and clear. It was very easy to read through.

Storyboard 8:
The storyboard could be a bit more clear on what the app solves. We found that we had to look at the storyboard title to understand what the app was for/doing. However, the app idea is pretty good.
